include/constants/battle_anim.h

```diff
@@ -592,5 +596,29 @@
 #define BACK_ANIM_FADE_GREEN_WITH_SHAKE        0x18
 #define BACK_ANIM_FADE_BLUE_WITH_SHAKE         0x19
 
+// mon pal blend
+#define PAL_BG 0x1
```

Comment:
Please add ANIM_ prefix to pal, fists, and surf_palette(which could be shortened to SURF_PAL)